spec/data/sig_handler.rb.txt

```diff
@@ -107,6 +107,10 @@ class SigReturn
 
   sig {void}
   def void_method; end
+
+  sig { void }
+  def method_definition_contains_comment # rubocop:disable ...
+  end
```

Comment:
We should test the other node types as well (`defs` and `command`). We might want to consider 3.0 `end`less defs as well, but probably better to take 3.0 support to a separate PR.